packages/backend/src/plugins/catalog.ts

```diff
@@ -34,9 +34,15 @@ export default async function createPlugin(
     locationAnalyzer,
   } = await builder.build();
 
+  const refreshLoopMs =
+    env.config.getOptionalNumber('backend.refreshLoopMs') || 100000;
```

Comment:
Umm, is this really needed to be read from app config? You can update this in code in your Backstage instance i.e. `packages/backend/.../catalog.ts`

Reply:
I don't think so. Everyone can adjust it in the own app, or introduce a config on their own if required. 

If we still add it, we should make sure to include the new `backend.refreshLoopMs` field in the config schema.

Reply:
Hmm yeah I think I'll vote to actually not put it in config at this time. It's not really necessary, since this is already at the very surface. If we start making the builder fire up the refresh loop and stuff, then maybe it would be a different matter. Then again - in that situation we might just as well make it a setter on the builder instead :)

Reply:
Happy to close this and edit our own backend. Thanks for the feedback all :)